Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -1934,7 +1935,9 @@ ExceptionOr<Ref<CanvasGradient>> CanvasRenderingContext2DBase::createConicGradie
     if (!std::isfinite(angleInRadians) || !std::isfinite(x) || !std::isfinite(y))
         return Exception { NotSupportedError };
 
-    return CanvasGradient::create(FloatPoint(x, y), angleInRadians, *this);
+    // Angle starts from x-axis for consistency within canvas methods https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-createconicgradientΩ
+    angleInRadians = normalizeAngle(angleInRadians) + piOverTwoFloat;
```

Comment:
What will happen if `normalizeAngle(angleInRadians) + piOverTwoFloat` is greater than 2 * pi? Should not the calculation be like this?

`angleInRadians = normalizeAngle(angleInRadians + piOverTwoFloat);`

Reply:
I'm addressing specifically the case where `angleInRadians` is potentially greater than `std::numeric_limits<float>::max() - π/2`, which could cause overflow as pointed by @pxlcoder. Normalizing after the sum would not help here.

CanvasGradient::create(..., angleInRadians,...) was already receiving non-normalized angles in radians, so I guess it is not an issue.

 I just want to make sure I can sum up without overflowing. But we can also normalize it again if you prefer.